{
  "Header": {
    "Original": "CVE-2022-1623 (LibTIFF master branch has an out-of-bounds read in LZWDecode in libtif ...)",
    "Line": 84131,
    "ID": "CVE-2022-1623",
    "Description": "(LibTIFF master branch has an out-of-bounds read in LZWDecode in libtif ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5333-1}",
      "Line": 84132,
      "Type": "xref",
      "Bugs": [
        "DSA-5333-1"
      ]
    },
    {
      "Original": "- tiff 4.4.0~rc1-1",
      "Line": 84133,
      "Type": "package",
      "Package": "tiff",
      "Kind": "fixed",
      "Version": "4.4.0~rc1-1"
    },
    {
      "Original": "[buster] - tiff \u003cnot-affected\u003e (Vulnerable code introduced later, PoCs don't trigger)",
      "Line": 84134,
      "Type": "package",
      "Release": "buster",
      "Package": "tiff",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later, PoCs don't trigger"
    },
    {
      "Original": "NOTE: https://gitlab.com/libtiff/libtiff/-/commit/b4e79bfa0c7d2d08f6f1e7ec38143fc8cb11394a (v4.4.0rc1)",
      "Line": 84135,
      "Type": "NOTE",
      "Description": "https://gitlab.com/libtiff/libtiff/-/commit/b4e79bfa0c7d2d08f6f1e7ec38143fc8cb11394a (v4.4.0rc1)"
    },
    {
      "Original": "NOTE: Introduced by: https://gitlab.com/libtiff/libtiff/-/commit/3079627ea0dee150e6a208cec8381de611bb842b (v4.4.0rc1)",
      "Line": 84136,
      "Type": "NOTE",
      "Description": "Introduced by: https://gitlab.com/libtiff/libtiff/-/commit/3079627ea0dee150e6a208cec8381de611bb842b (v4.4.0rc1)"
    },
    {
      "Original": "NOTE: https://gitlab.com/libtiff/libtiff/-/issues/410",
      "Line": 84137,
      "Type": "NOTE",
      "Description": "https://gitlab.com/libtiff/libtiff/-/issues/410"
    }
  ]
}